sample/host.json

```diff
@@ -24,12 +24,19 @@
             "maxOutstandingRequests": 30
         },
         "queues": {
-            "visibilityTimeout": "00:00:10"
+            "visibilityTimeout": "00:00:10",
+            "maxDequeueCount": 3
         },
-        "eventHub": {
+        "eventHubs": {
             "maxBatchSize": 1000,
             "prefetchCount": 1000,
             "batchCheckpointFrequency": 1
+        },
+        "serviceBus": {
+            "prefetchCount": 100,
+            "messageHandlerOptions": {
```

Comment:
Just including an example showing the new config hierarchy, tested also in PR https://github.com/Azure/azure-webjobs-sdk/pull/1856